src/common/chat/autocompleter.ts

```diff
@@ -22,9 +24,20 @@ export class Autocompleter {
       const char = entry.charAt(i);
       const maybeChild = currentNode.getChild(char);
       currentNode =
-          maybeChild !== undefined ?
-              maybeChild : currentNode.addValueAsNeighbor(char);
+        maybeChild !== undefined ?
+          maybeChild : currentNode.addValueAsNeighbor(char);
     }
+
+    const tokens = option.split(' ');
+    tokens.forEach((token) => {
+      if (!this.tokenMap.has(token)) {
+        this.tokenMap.set(token, [option]);
+      } else {
+        const curOptions: string[] = checkDefined(this.tokenMap.get(token));
```

Comment:
this.tokenMap.get(token).push(option) work? I would think it's the same object reference right?

Reply:
It does, and that's cleaner. Doing it implicitly can get a little confusing, so I just threw a comment in